proto/ethermint/evm/v1/genesis.proto

```diff
@@ -22,7 +22,9 @@ message GenesisAccount {
   string address = 1;
   // code defines the hex bytes of the account code.
   string code = 2;
+  // codeHash defines the keccak256 hash of the hex bytes of the account code.
+  string codeHash = 3;
```

Comment:
I don't get it, isn't it always `keccak256(code)`, maybe I have some misunderstanding here.

Reply:
That codeHash will not be set at this point in time. It does not get set at genesis currently.

Reply:
when you export a genesis, the codeHash under the auth section should be correct.
I don't get what's the issue we are trying to solve here?

Reply:
Motivation for this PR was to add precompiled contracts to the chain before the first block is produced.

Reply:
I see, I've done sth like this before, what I do is start a local devnet, deploy the contract, and export the genesis file.
It should be possible to directly modify the genesis file too, what's the obstacle of doing that right now?

Reply:
https://github.com/crypto-org-chain/cronos/blob/main/integration_tests/configs/genesis_token_mapping.yaml#L44

Reply:
![image](https://user-images.githubusercontent.com/93690142/158007915-e88b4c5d-54ff-462c-91fd-f3184683f80b.png)